_includes/footer/footer-en.html

```diff
@@ -23,9 +24,35 @@
     </div>
     <div id="footer-links">
       <div class="footer-social">
-        <div id="github">
-          <a href="https://github.com/expressjs/express" aria-label="Go to the repository on GitHub"><i class="fa fa-github"></i></a>
-        </div>  
+        <div>
+          <a href="https://github.com/expressjs/express" aria-label="Go to the repository on GitHub">
+            <span class="hidden-dark">
+              {% include icons/github-light.svg %}
+            </span>
+            <span class="hidden-light">
+              {% include icons/github-dark.svg %}
+            </span>
+          </a>
+        </div>
+        <div>
+          <a href="https://www.youtube.com/channel/UCYjxjAeH6TRik9Iwy5nXw7g" aria-label="Go to the repository on Youtube">{% include icons/youtube.svg %}</a>
+        </div>
+        <div>
+          <a href="https://x.com/UseExpressJS" aria-label="Go to the repository on X">
+            <span class="hidden-dark">
+              {% include icons/x-light.svg %}
+            </span>
+            <span class="hidden-light">
+              {% include icons/x-dark.svg %}
+            </span>
+          </a>
+        </div>
+        <div>
+          <a href="https://openjs-foundation.slack.com/archives/C02QB1731FH" aria-label="Go to the repository on Slack">{% include icons/slack.svg %}</a>
```

Comment:
```suggestion
          <a href="https://slack-invite.openjsf.org/" aria-label="Go to the repository on Slack">{% include icons/slack.svg %}</a>
```

Maybe is better to provide a link to the slack invitation rather than the channel? Ideally we can a small section in the web where we explain how to join slack and what channels to monitor. :thinking:

Reply:
I think we'd prefer them to join the specific Slack channel instead of just Slack itself, as Slack is more general and includes all OpenJSF projects. The idea here is to strengthen Express

Reply:
@expressjs/docs-wg, what do you all think about this?

Reply:
I agree with @bjohansebas , we should direct to the channel for the express